ubuntu-packages.txt

```diff
@@ -1,6 +1,8 @@
 clang-format
-enchant
+enchant-2
 git
 python3
 python3-pip
 python3-venv
+python3-sphinx
+python3-sphinxcontrib.spelling
```

Comment:
Normally, these would come from here: https://github.com/KhronosGroup/SYCL_Reference/blob/8802b5beb5bb2096c15ad48dda14a2bcfb6be998/requirements.txt#L6
when you do 
```
pip3 install -r requirements.txt
```

I always work in a virtual environment and prefer not to install system packages. Are you OK with not adding these 2 here?

Reply:
Yes, no problem.